arches/app/models/models.py

```diff
@@ -1577,9 +1577,11 @@ class Meta:
 
 class WorkflowHistory(models.Model):
     workflowid = models.UUIDField(primary_key=True)
+    workflowname = models.TextField(null=True, default=None)
```

Comment:
ah -- the purpose of adding null=True is to support migrations. If a user has WorkflowHistory objects already the migration will fail to run without this flag. I went ahead and changed it to a CharField

Reply:
Interesting. I usually let the interactive questioner catch things like that.
```py
It is impossible to add a non-nullable field 'newfield' to workflowhistory without specifying a default. This is because the database needs something to populate existing rows.
Please select a fix:
 1) Provide a one-off default now (will be set on all existing rows with a null value for this column)
 2) Quit and manually define a default value in models.py.
```

What about a default of `""`?

Reply:
to me a nullable field is more intentional/descriptive than a field that defaults to an empty string

Reply:
that's fair 👍 

I just like finding out what people's preferences on this one, as it comes up a good bit. this looks ready to merge!